h2o-py/h2o/h2o.py

```diff
@@ -1257,6 +1259,102 @@ def make_metrics(predicted, actual, domain=None, distribution=None):
               data={"domain": domain, "distribution": distribution})
     return res["model_metrics"]
 
+gen_model_file_name = "h2o-genmodel.jar"
```

Comment:
these should probably not be defined here (they have no global significance)

Reply:
moved it to shared_utils.py